### PR TITLE
(iOS) Fix keyboardAppearance in dark mode

### PR DESCRIPTION
**Pull request by moubaraks**
Fixes the keyboard's dark mode appearance

### CODE REVIEW

**Review by bugnano — CHANGES_REQUESTED** [review]
Just one thing: Instead of merging this PR into `master`, you should merge it into the `non_beta_build` branch instead.

File: lib/src/in_app_webview/in_app_webview.dart [hunk 1]
```diff
@@ -787,6 +787,7 @@ class _InAppWebViewState extends State<InAppWebView> {
           'initialSettings': settingsMap,
           'contextMenu': widget.contextMenu?.toMap() ?? {},
           'windowId': widget.windowId,
+          'keyboardAppearanceDark': MediaQuery.platformBrightnessOf(context) == Brightness.dark,
```
Comment (vickz84259) [inline lib/src/in_app_webview/in_app_webview.dart]: I've tested the change and it doesn't fix the issue highlighted [here](https://github.com/talkjs/talkjs-flutter/issues/24). Here are steps to reproduce:

1. Inside `main.dart` add theme: `ThemeData.dark()` to `MaterialApp`'s constructor.
2. Make sure the iPhone's appearance setting under display & brightness is set to light
3. Run app. The app will be in dark mode but the keyboard will still be the light mode version.

By default, the iOS keyboard switches between light and dark mode in respective to the current appearance setting in Settings. So using `MediaQuery.platformBrightnessOf` doesn't fix the issue. Instead use `Theme.of` to determine the current theme the app is in and choose the light or dark keyboard appearance accordingly.

**Review by vickz84259 — CHANGES_REQUESTED** [review]
The change currently switches keyboard appearance based on the OS's appearance setting. However the OS already does this by default. The change needs to use whatever theme appearance the app is currently using since most apps allow their users to explicitly choose dark or light or for it to use the current system setting.

File: lib/src/in_app_webview/in_app_webview.dart (hunk 1 above)
Comment (moubaraks) [reply lib/src/in_app_webview/in_app_webview.dart]: Fixed

**Review by vickz84259 — COMMENTED** [review]
Awesome work @moubaraks.

**Pull request merged**